apps/server/lib/lexical/server/code_intelligence/completion/translations/callable.ex

```diff
@@ -118,11 +128,35 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Callable do
 
   @default_functions ["module_info", "behaviour_info"]
 
-  defp maybe_boost(item, %_{name: name}, default_boost \\ 5) do
-    if String.starts_with?(name, "__") or name in @default_functions do
-      item
-    else
-      Builder.boost(item, default_boost)
+  defp maybe_boost(item, callable, %Env{position_module: position_module}) do
```

Comment:
nit: i prefer using function-level pattern matching to make decisions about which function head is called, and not to extract fields. 
```suggestion
  defp maybe_boost(item, callable, %Env{}) do
  position_module = env.position_module
```

Reply:
Fixed!